auslib/web/public/base.py

```diff
@@ -33,9 +33,9 @@
 
 current_dir = path.dirname(__file__)
 web_dir = path.dirname(auslib.web.__file__)
-spec = SpecBuilder().add_spec(path.join(current_dir, 'api.yml'))\
-                    .add_spec(path.join(web_dir, 'common/releases_spec.yml'))\
-                    .add_spec(path.join(web_dir, 'common/rules_spec.yml'))
+spec = SpecBuilder().add_spec(path.join(current_dir, 'swagger/api.yml'))\
+                    .add_spec(path.join(current_dir, 'swagger/public_api_spec.yml'))\
+                    .add_spec(path.join(web_dir, 'common/swagger/common_spec.yml'))
```

Comment:
I'd like to try to come up with a better name for this than "common_spec.yml". The current name makes it very opaque as to what is in it. Maybe there's a term we can find that encompasses "parameters, definitions, responses"? Or maybe we should split out each section into its own file?